packages/drivers/odsp-driver/src/odspDriverUrlResolver2.ts

```diff
@@ -22,10 +22,17 @@ export class OdspDriverUrlResolver2 implements IUrlResolver {
         private readonly identityType: IdentityType = "Enterprise",
         private readonly logger?: ITelemetryLogger,
         private readonly appName?: string,
+        private readonly containerPackageName?: string,
```

Comment:
I don't think we want to supply container package name in resolver constructor. That would mean that this resolver will be tied to particular container package.

Reply:
Yeah package name and resolver aren't 1:1, package name and quorum information/container should be the 1:1 relationship here. Also where is this getting passed in from?

Reply:
I imagined this would come from quorum, not from host code which doesn't have a reason to need to parse IFluidCodeDetails and actually won't have IFluidCodeDetails outside of quorum in regular load scenarios...

Reply:
thanks for the comments, I'm getting rid of this since this is already merged #4001